releasing/build.sh

```diff
@@ -18,5 +18,4 @@ set -e
 
 cd $(dirname "$0")/..
 
-source ./.github/bin/github-releases-setup.sh /out
-./.github/bin/github-pages-setup.sh $PREFIX_BIN
+./.github/bin/github-releases-setup.sh /out
```

Comment:
Why do we need to create a `/out`, which we might not have permission to, vs. the default `-/tmp/releases` we get without the parameter ?

Reply:
This `/out` path is also referenced in https://github.com/chipsalliance/verible/blob/master/releasing/docker-run.sh#L136.